tests/src/readytorun/mainv2.csproj

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
MainV1 and V2 are identical except for the version of the test dll they attempt to load. The test build system generates the .cmd files that launch the tests and the content can be customized in the .csproj for the exe of the test. Moreover, the launch of the test exe is also not customizable in a way that would allow me to add a parameter to the exe that would specify which .dll to load.
So there doesn't seem to be a way to use the same main.exe for both tests. And since we need to generate two exes, the way I have used seemed convenient.
Does it make sense?